src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -204,6 +289,15 @@ namespace Microsoft.FSharp.Collections
         /// <param name="source">The input sequence.</param>
         ///
         /// <returns>True if the input sequence contains the specified element; false otherwise.</returns>
+        ///
+        /// <example>
```

Comment:
Give each `<example>` a unique id, see #12134 for examples

Reply:
Solved